cmd/gotagger/main.go

```diff
@@ -79,6 +80,17 @@ func (g *GoTagger) Run() int {
 	flags.BoolVar(&g.tagRelease, "release", g.boolEnv("release", false), "tag HEAD with the current version if it is a release commit")
 	flags.StringVar(&g.versionPrefix, "prefix", g.stringEnv("prefix", "v"), "set a prefix for versions")
 	flags.StringVar(&g.dirtyIncrement, "dirty", g.stringEnv("dirty", ""), "how to increment the version for a dirty checkout [minor, patch]")
+	flags.StringVar(&g.configFile, "config", g.stringEnv("config", ""), "path to the gotagger configuration file.")
+
+	if g.configFile == "" {
+		// If there's no config file provided, check for one locally.
+		defaultConfig := filepath.Join(g.WorkingDir, "gotagger.json")
+		_, err := os.Stat(defaultConfig)
+		if err == nil {
+			g.configFile = defaultConfig
+			log.Println("using config file " + defaultConfig)
```

Comment:
Delete this, it always outputs and gotagger's only output should be the version.